myhoard/controller.py

```diff
@@ -880,7 +880,8 @@ def _current_normalized_backup_timestamp(self) -> str:
         else:
             if normalized.hour < backup_hour or (normalized.hour == backup_hour and normalized.minute < backup_minute):
                 normalized = normalized - datetime.timedelta(days=1)
-            normalized = normalized.replace(hour=backup_hour, minute=backup_minute, second=0, microsecond=0)
+        
+        normalized = normalized.replace(hour=backup_hour, minute=backup_minute, second=0, microsecond=0)
```

Comment:
I think you need to pull the whole if statement out of the else, as we need to do the date math to make sure our window isn't > 24h.

Reply:
It also would be worth excavating if the actual bug is just that `_previous_normalized_backup_timestamp` parses all streams, rather than just active basebackup ones. I.e. I expect the reason we check the old one is so we don't change the request time while a stream is active, but then we get stuck in this perma-loop. I think we should just remove the state checking behaviour, just would need to validate how things behave when we change this in the middle of a backup (would not want to abort).

Reply:
It needs to check the previous backup time because if the backup interval is greater than 24 hours the previous logic will still run the backup at least once a day.

I started trying to solve the problem more generally but loads of problems arise because the interval is in minutes so the time drifts every day making the backup hour/minute number kind of meaningless.

I think maybe the simplest but not so nice way to fix our immediate problem with this is to check if the interval is greater than 24 hours and use the current logic to allow intervals greater than 24 hours, otherwise use the old logic so you can change the backup time.

Otherwise if we enforced some kind of allowed values on the interval, like hourly then if you get past 24 hours you can only have values that have 24 hours as a factor this all becomes a lot more sensible.

Reply:
Makes sense to me. Can just update the documentation to say for intervals != 24h, the backup hour/minute configuration is not respected, as even if its less than 24hrs it doesn't really make sense either.

Reply:
Ideally we would reject configuration that sets both sets to non-defaults but that would be a breaking change now I guess.